f3.go

```diff
@@ -39,13 +38,12 @@ type f3State struct {
 	ps       *powerstore.Store
 	certsub  *certexpoll.Subscriber
 	certserv *certexchange.Server
-	manifest *manifest.Manifest
 }
```

Comment:
I left the `f3State` behind to keep the internal pause/resume working - used solely for testing.